cloud/jenkins/openshift-cluster-destroy.yml

```
@@ -0,0 +1,60 @@
+          description: 'Reason for cluster destruction'
+      - bool:
+          name: CLEANUP_REMAINING
+          default: true
+          description: 'Attempt to clean up any remaining resources after destruction'
+    
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/Percona-Lab/jenkins-pipelines.git
+            branches:
+            - 'master'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 14 but found 12

(indentation)